util/src/main/java/tk/zielony/randomdata/util/StringCarGenerator.java

```diff
@@ -0,0 +1,23 @@
+package tk.zielony.randomdata.util;
+
+import android.support.annotation.Nullable;
+
+import java.util.Random;
+
+import tk.zielony.randomdata.DataContext;
+import tk.zielony.randomdata.Generator;
+
+/**
+ * Created by kirinpatel on 12/1/17.
+ */
+
+public class StringCarGenerator extends Generator<String> {
```

Comment:
This generator could be replaced with a generic enum generator. Cars are a broad topic. Type of car body could come with model, manufacturer names, engines, etc. in a form of a data base. All of these attributes should be linked using DataContext.